packages/clarity-js/src/data/metadata.ts

```diff
@@ -19,19 +19,29 @@ export function start(): void {
   data = {
     projectId: config.projectId || hash(location.host),
     userId: user(),
-    sessionId: s.id,
+    sessionId: s.session,
     pageNum: s.count
   }
 
+  // For backward compatibility; remove in future iterations (v0.6.11)
+  if (config.upload && typeof config.upload === Constant.String && (config.upload as string).indexOf(Constant.HTTPS) === 0) {
+    let url = config.upload as string;
+    config.server = url.substr(0, url.indexOf("/", Constant.HTTPS.length));
+    config.upload = config.server.length > 0 && config.server.length < url.length ? url.substr(config.server.length + 1) : url;
+  }
```

Comment:
any reason why we cant just keep track of clarity-js version in the session storage for the backward forward compatibility - if the version is not-updated you can send to old session DC else send to new one (and reset session?)